frontend/src/components/Election/Results/RoundResultTabs.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {useState} from 'react'
+import Typography from '@mui/material/Typography';
+import {Paper, Box} from '@mui/material';
+
+const RoundResultTabs = ({children, rounds}) => {
+    const [currentTab, setCurrentTab] = useState(0);
+
+    if(rounds == 1) return <>{children}</>
+
+    let i = 0;
+    const roundIndexes = Array.from({length: rounds}, () => i++);
+
+    return <>
+        {rounds >= 5 && 
+            <Typography variant="h6">Rounds</Typography>
```

Comment:
We might want to use a different word than Rounds here. While we use it in the code as one winner per round some people might get it confused with score/runoff rounds.

Reply:
Agreed, changed it to winner in the latest update